dao2caom2/tests/test_composable.py

```diff
@@ -73,6 +73,14 @@
 from mock import Mock, patch
 from dao2caom2 import composable, dao_name, COLLECTION
 
+F_NAME_LIST = [
+    'data_report.txt',
+    'failure_log.txt',
+    'rejected.yml',
```

Comment:
Why this has a different format?

Reply:
`*.txt` files are logging and for output only. `*.yml` are for state information, and are mostly read/write. `rejected.yml` is read/write.

There's no technical reason the `.txt` files could not also be `.yml` files, but it makes it more straight-forward to clean up transient information after a run: `rm *.txt` is mindless, and usually harmless. The way the files names are currently assigned `rm *.yml` is an "oh no second".

If you think the files should be named/handled more consistently, let me know, and I'll figure something out.